### ISSUE TITLE
Trackback: Force resending trackback/pingback

### ISSUE THREAD

**Issue by onli** [s9y/Serendipity#440]
The trackback-plugin offers some options for controlling trackbacks, like disabling sending them altogether or sending them to a specific URL. But it seems it misses a way to say "Resend trackback to this link" or "Resend trackback to all links in entry".  Most user will never need it, but for debugging server issues it would be good to have this options.

It is possible that will need a core change to give the plugin the option to resend trackbacks that are already stored in the table `serendipity_references`.

**Comment by garvinhicking** [comment]
You can do that with the plugin by removing all trackback links, saving, pasting in the old trackback links, saving again.

**Comment by onli** [comment]
Good to know, thanks :) But a button to do it directly still would be helpful.

**Comment by onli** [comment]
This is basically solved now. I added code to the core to skip the check whether the trackbacks are already sent, and the trackbacks-plugin now has a checkbox to enable that skip. But because of the core change I did not add the new version to spartacus, but to its own branch. To be merged after the next release. PR is in https://github.com/s9y/additional_plugins/pull/116, so I think we don't need this issue anymore.
